smithy-aws-protocol-tests/model/restJson1/http-headers.smithy

```diff
@@ -265,7 +265,6 @@ structure InputAndOutputWithHeadersIO {
 /// Null and empty headers are not sent over the wire.
 @readonly
 @http(uri: "/NullAndEmptyHeadersClient", method: "GET")
-@tags(["client-only"])
 operation NullAndEmptyHeadersClient {
```

Comment:
Can these operations be consolidated now that the distinction is on the tests themselves? This would require removing the operation level tags, though, and possibly cause a downstream issue. Maybe we just consolidate this as a followup.

Reply:
Since I'm adding tags back for now, this will need to wait. But it's a good idea.